File: JAVA/Shirt.java
/* Saya Rifa Sania NIM 2206697 mengerjakan Latihan Praktikum 2 dalam mata kuliah DPBO
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin... */

class Shirt extends Clothing{
    private String color;
    private String sleeve_type;

    Shirt(){        
    }

    public String getColor(){
        return this.color;
    }

    public void setColor(String color){
        this.color = color;
    }

    public String getSleeve_type(){
        return this.sleeve_type;
    }

    public void setSleeve_type(String sleeve_type){
        this.sleeve_type = sleeve_type;
    }
}

File: CPP/Program/Shirt.cpp
/* Saya Rifa Sania NIM 2206697 mengerjakan Latihan Praktikum 2 dalam mata kuliah Desain Pemrograman Berorientasi Objek
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin... */

// include library
#include "Clothing.cpp"

class Shirt : public Clothing{
    // atribut
    private:
        string color;
        string sleeveType;

    public:
        Shirt(){
            // konstruktor kosong
        }

        // set dan get

        void setColor(string color){
            this->color = color;
        }

        string getColor(){
            return this->color;
        }

        void setSleeveType(string sleeveType){
            this->sleeveType = sleeveType;
        }

        string getSleeveType(){
            return this->sleeveType;
        }
};